src/account_multisig.ts

```diff
@@ -314,7 +314,54 @@ export class Account2FA extends AccountMultisig {
         }
     }
 
-    async disable(contractBytes: Uint8Array, cleanupContractBytes: Uint8Array) {
+    async disableWithFAK({ contractBytes, cleanupContractBytes }: { contractBytes: Uint8Array, cleanupContractBytes?: Uint8Array }) {
+        let cleanupActions = [];
+        const keyConversionActions = await this.get2faDisableKeyConversionActions();
```

Comment:
1. Let's move this call to after the `if(cleanupContractBytes){...}` block so that we don't hit RPC until we're sure that we haven't encountered a fatal error
2. Since this method will be used in cases where the multisig contract is unhappy/unusable, we _don't_ want to throw an error just because `deleteAllRequests()` failed; in this case it is acceptable for that attempt to fail, and if it does fail, we want to _ignore_ the failure and continue with removing the smart contract and re-converting their keys from LAKs to FAKs -- otherwise this method won't allow someone to disable 2FA for an account where the multisig contract is unusable for whatever reason (underlying state etc.).

Reply:
It's currently swallowing any `deleteAllRequests` errors and throwing them if `get2faDisableCleanupActions` throws but updated so it swallows all `deleteAllRequests` errors and throws any `get2faDisableCleanupActions` errors directly 👍